linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/js/process_info.js

```diff
@@ -19,7 +19,7 @@ var ProcInfo = (function() {
 
   function render(data) {
     _(data).each(function(obj) {
-      var id = obj.name.replace(/\//g, "-");
+      var id = obj.name.replace(/\/|\$/g, "-");
```

Comment:
Given that we might have more characters that we eventually want to replace here, I think it's more concise to use a bracket expression:

    var id = obj.name.replace(/[/$]/g, "-");

And that said, how about also escaping "."? I know that sometimes causes problems for jquery selectors. 

    var id = obj.name.replace(/[/$.]/g, "-");